### ISSUE TITLE
Duplicate configuration option for /etc/sysconfig/docker

### ISSUE THREAD

**Issue by brunopereira81** [clustervision/trinity#614]
The Trinity [postinstall script](https://github.com/clustervision/trinity/blob/r7/controller/rootimg/install/custom/netboot/centos/trinity.centos7.postinstall#L27) adds the same option in to your custom docker [file](https://github.com/clustervision/trinity/blob/r7/controller/rootimg/etc/sysconfig/docker#L4) resulting in :

```
[root@node003 ~]# cat /etc/sysconfig/docker
# /etc/sysconfig/docker

# Modify these options if you want to change the way the docker daemon runs
OPTIONS=--insecure-registry controller:5050 --insecure-registry controller:5050 --bip=10.146.255.254/16 --fixed-cidr=10.146.255.255/16 '--selinux-enabled'
`

**Comment by hansthen** [comment]
Good catch.

**Comment by Marc69** [comment]
Solved in * r8 3bb0b2d (tmaster)
```
[root@node001 ~]# cat /etc/sysconfig/docker
# /etc/sysconfig/docker

# Modify these options if you want to change the way the docker daemon runs
OPTIONS=--insecure-registry controller:5050 --bip=10.146.255.254/16 --fixed-cidr=10.146.255.255/16 --selinux-enabled
```
